src/Ext/freecad/CMakeLists.txt

```diff
@@ -10,7 +10,11 @@ if (WIN32)
                        REALPATH BASE_DIR "${CMAKE_INSTALL_PREFIX}")
     set( ${CMAKE_INSTALL_BINDIR})
```

Comment:
can someone explain what this does?

Reply:
This sets the path to the install folder of CMake. In my case to
`C:\Program Files\CMake\bin`

I think it is there for the case that users installed CMake without adding it to the global PATH variable. In this case thanks to CMAKE_INSTALL_BINDIR, the CMake executable is found nevertheless. Therefore It doesn't harm to keep it.

Reply:
I don't think it's related to the cmake installation, `CMAKE_INSTALL_BINDIR` is where the compiled binaries will get installed, in my case `/usr/lib/freecad/bin`
Looking at the [cmake documentation for the set command](https://cmake.org/cmake/help/latest/command/set.html) this line makes no sense, when you provide a sinlge argument it is supposed to be a variable identifier that would be unset, if you pass `${CMAKE_INSTALL_BINDIR}` I think it gets evaluated so it is actually receiving just a path as argument, which makes no sense. Plus I don't see why you would want to unset `CMAKE_INSTALL_BINDIR`